Escaped characters in error messages.

When I use the configuration option `needs_sphinx` and set the value to a Sphinx version I don't have installed, I get an error message (in german) that contains escaped characters, i.e. I get:

```
Sphinx version error:
Dieses Projekt ben\xf6tigt Version 3.1.0 oder sp\xe4ter und kann daher nicht gebaut werden.
make: *** [html] Error 2
```

I would have expected:

```
Sphinx version error:
Dieses Projekt benötigt Version 3.1.0 oder später und kann daher nicht gebaut werden.
make: *** [html] Error 2
```

My Sphinx version is 2.1.2, Python is

```
Python 3.7.3 (default, Apr 24 2019, 11:03:12)
[Clang 10.0.1 (clang-1001.0.46.4)] on darwin
```

and the environment variable `PYTHONIOENCODING` is set to `utf-8`.

--- a/CHANGES
+++ b/CHANGES
@@ -49,6 +49,7 @@ Bugs fixed
 * #6511: LaTeX: autonumbered list can not be customized in LaTeX
   since Sphinx 1.8.0 (refs: #6533)
 * #6531: Failed to load last environment object when extension added
+* #6549: sphinx-build: Escaped characters in error messages
 
 Testing
 --------

--- a/sphinx/cmd/build.py
+++ b/sphinx/cmd/build.py
@@ -47,7 +47,7 @@ def handle_exception(app: Sphinx, args: Any, exception: BaseException, stderr: I
             print(terminal_safe(exception.args[0]), file=stderr)
         elif isinstance(exception, SphinxError):
             print(red('%s:' % exception.category), file=stderr)
-            print(terminal_safe(str(exception)), file=stderr)
+            print(str(exception), file=stderr)
         elif isinstance(exception, UnicodeError):
             print(red(__('Encoding error:')), file=stderr)
             print(terminal_safe(str(exception)), file=stderr)

